File: scripts/postbuild.py
#!/usr/bin/env python3
"""Post-build script: copy old Lovable assets + operacoes.html into dist"""
import shutil, os

PUBLIC = 'public'
DIST = 'dist'
ASSETS = os.path.join(DIST, 'assets')

# 1. Copy old Lovable bundle and CSS into dist/assets/
shutil.copy2(os.path.join(PUBLIC, 'assets', 'index-Cdosnuq2.js'), ASSETS)
shutil.copy2(os.path.join(PUBLIC, 'assets', 'index-DBm4vQ15.css'), ASSETS)

# 2. Copy old Lovable index.html (overwrites Vite's generated one)
shutil.copy2(os.path.join(PUBLIC, 'lovable-index.html'), os.path.join(DIST, 'index.html'))

# 3. Copy operacoes.html
shutil.copy2(os.path.join(PUBLIC, 'operacoes.html'), os.path.join(DIST, 'operacoes.html'))

print("Post-build: old Lovable assets + operacoes.html copied to dist/")

File: scripts/postbuild.mjs
// Build script: this site has no bundler step — it's the old Lovable
// export (Aquisição, at "/") plus a standalone HTML app (Operações, at
// "/operacoes"), both served as static files. This just assembles dist/.
import fs from 'node:fs';
import path from 'node:path';

const PUBLIC = 'public';
const DIST = 'dist';

fs.rmSync(DIST, { recursive: true, force: true });
fs.mkdirSync(DIST, { recursive: true });
fs.cpSync(PUBLIC, DIST, { recursive: true });

// Lovable's own index.html is the site's real entry point at "/".
fs.renameSync(path.join(DIST, 'lovable-index.html'), path.join(DIST, 'index.html'));

console.log('Build: public/ copied to dist/ (lovable-index.html -> index.html)');
